apps/web/src/common/modules/navigations/top-bar/modules/top-bar-header/TopBarWorkspaces.vue

```diff
@@ -130,6 +133,15 @@ const handleClickAllWorkspaceButton = () => {
         name: LANDING_ROUTE._NAME,
     });
 };
+
+watch(() => storeState.selectedWorkspace, (selectedWorkspace) => {
+    if (!selectedWorkspace) return;
+    recentStore.createRecent({
+        type: RECENT_TYPE.WORKSPACE,
+        workspaceId: selectedWorkspace?.workspace_id || '',
+        id: selectedWorkspace?.workspace_id || '',
+    });
```

Comment:
nitpick: It would be more reasonable not to create recent without `workspace_id`!